cmd/activator/main.go

```diff
@@ -236,7 +237,7 @@ func main() {
 
 	// NOTE: MetricHandler is being used as the outermost handler of the meaty bits. We're not interested in measuring
 	// the healthchecks or probes.
-	ah = activatorhandler.NewMetricHandler(env.PodName, ah)
+	ah = activatorhandler.NewMetricHandler(env.PodName, networkConfig.DataplaneTrust, ah)
```

Comment:
this might be a little outdated, I think the activator was changed recently to dynamically pick up config changes?  I haven't looked over those changes in regards to this PR yet.

https://github.com/knative/serving/issues/13694